components/dashboard/tailwind.config.js

```diff
@@ -50,6 +50,9 @@ module.exports = {
                 // TODO(andreafalzetti): remove custom ide-modal class once we implement https://github.com/gitpod-io/gitpod/issues/13116
                 51.5: "51.5rem",
             },
+            lineHeight: {
```

Comment:
Adding this allows us to use a `leading-64` class on our `h1` so that it has the right `line-height`.